src/ui/templates/insurance/check-your-answers.njk

```diff
@@ -0,0 +1,76 @@
+{% extends 'index.njk' %}
+
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+
+{% block pageTitle %}
+  {{ CONTENT_STRINGS.PAGE_TITLE }}
+{% endblock %}
+
+{% block content %}
+
+  {{ govukBackLink({
+    text: CONTENT_STRINGS.LINKS.BACK,
+    href: BACK_LINK,
+    attributes: {
+      "data-cy": "back-link"
+    }
+  }) }}
+
+  {% set html %}
+    <p class="govuk-notification-banner__heading" data-cy="change-eligibility-banner">{{ CONTENT_STRINGS.CHANGE_ELIGIBILITY }} <a class="govuk-notification-banner__link" href="{{ START_NEW_APPLICATION }}" data-cy="change-eligibility-link">{{ CONTENT_STRINGS.CHANGE_ELIGIBILITY_LINK }}</a></p>
+  {% endset %}
+
+  {% if notificationBanner %}
```

Comment:
I don't think we need this conditional here or in the controller as it'll always be true?

Reply:
the other pages don't have the banner

Reply:
Oh yes sorry I forgot this is a generic answers template (brilliant) - can we rename this to `renderBanner`? Other places use `renderSuccessBanner`